### PR TITLE
add Ondřej Bojar and DFKI

### LINKED ISSUE #169: Article: Ondřej Bojar

### PR DESCRIPTION
**Pull request by zouharvi**
# Description

Fixes #169 (Ondřej Bojar) and lists DFKI, a potent German research institute, in research laboratories.
It also reorders the list to be alphabetically sorted.

## Type of PR

- Creates the article _Ondřej Bojar_
- Edits _Research laboratories_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).
- [x] I have made sure that the URL is not already in use.
- [x] I have cross-linked relevant articles.
- [x] I have used the UK spelling.
- [x] I have kept consistency with the structure of sister articles in the same directory.

### CODE REVIEW

File: people/ondrej-bojar.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+grand_parent: Community
+parent: People
+title: Ondřej Bojar
+description: Machine translation researcher
```
Comment (cefoo) [inline people/ondrej-bojar.md:5]: Hi Vilém! 

Thank you for the PR!

Other `People` pages (like [Alon Lavie](https://github.com/machinetranslate/machinetranslate.org/blame/master/people/alon-lavie.md) include a wider `description`, but I see inconsistencies among these pages.  Do you think it's best to include more info, ie. `Machine translation researcher and associate professor`, in all these pages?

File: people/ondrej-bojar.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+**Ondřej Bojar** is a researcher and associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague, widely cited for his work on statistical and later [neural machine translation](/neural-machine-translation).
```
Comment (cefoo) [inline people/ondrej-bojar.md:8]: I would separate this sentence into two:

> He is widely cited for his work on [statistical](/approaches/statistical-machine-translation.md) and [neural machine translation](/approaches/neural-machine-translation.md).

File: people/ondrej-bojar.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+He co-authored Moses, a kit for [statistical machine translation](/statistical-machine-translation) and is a long-term organizer of [WMT](/community/communities.md#workshop_on_statistical_machine_translation).
```
Comment (cefoo) [inline people/ondrej-bojar.md:9]: I guess if we link `statistical machine translation` in the previous paragraph, we can avoid it here.

We have a new page for **WMT**, which was reviewed by Barry Haddow!😄 You can link **WMT** to `/events/wmt.md`.

File: people/ondrej-bojar.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+His work goes beyond mainstream machine translation, such as interactive machine translation, post-editing or psycholinguistical aspects of machine translation usage.
+He managed Charles University's participation in the [Bergamot](https://browser.mt/) project for browser-based machine translation and the [ELITR](https://elitr.eu/) project for simultaneous translation of European languages and is the author of [CzEng](https://ufal.mff.cuni.cz/czeng), the Czech-English parallel corpus with over 60 million authentic sentence pairs.
```
Comment (cefoo) [inline people/ondrej-bojar.md:12]: I would also separate this sentence into two for better performance when we machine translate the site. 😉

File: people/ondrej-bojar.md (hunk 2 above)
Comment (zouharvi) [reply people/ondrej-bojar.md:8]: Good point.

File: people/ondrej-bojar.md (hunk 4 above)
Comment (zouharvi) [reply people/ondrej-bojar.md:12]: It doesn't look nice stylistically to have so many sentences start with "He ..." but I see your point.

File: people/ondrej-bojar.md (hunk 3 above)
Comment (zouharvi) [reply people/ondrej-bojar.md:9]: I overlooked that, thanks. :-)

File: people/ondrej-bojar.md (hunk 1 above)
Comment (zouharvi) [reply people/ondrej-bojar.md:5]: I don't have a strong opinion on this. "Associate/assistant" professors tend to change titles often so we need to keep up with that.

**Comment by zouharvi** [comment]
Hi @cefoo, thank you for your review. I agree with you on all the points and updated the PR accordingly. :slightly_smiling_face:

**Comment by zouharvi** [comment]
Hi, sorry it took so long (lots of work duties in the meantime). I incorporated your changes, though I'm unsure how to make the bio more evergreen since OB didn't start the MT group (AFAIK) and it's not organized into "groups" here anyway.

File: people/ondrej-bojar.md [hunk 5]
```diff
@@ -5,8 +5,8 @@ title: Ondřej Bojar
 description: Machine translation researcher and associate professor
 ---
 
-**Ondřej Bojar** is a machine translation researcher and since 2010 an associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague.
-His work on [statistical](/statistical-machine-translation) and later [neural machine translation](/neural-machine-translation) is widely cited.
+**Ondřej Bojar** is a machine translation researcher at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague.
+He became an associate professor in 2019 and his work on [statistical](/statistical-machine-translation) and later [neural machine translation](/neural-machine-translation) is widely cited.
```
Comment (cefoo) [inline people/ondrej-bojar.md:9]: I think the link should be as follows:

`and his work on [statistical](/../approaches/statistical-machine-translation.md) and later [neural machine translation](/../approaches/neural-machine-translation.md)...`

We have created an issue #238 to fix links, but for the time being, this is the way they are working.

File: people/ondrej-bojar.md [hunk 6]
```diff
@@ -5,8 +5,8 @@ title: Ondřej Bojar
...
 He co-authored [Moses](http://www2.statmt.org/moses/), a system for statistical machine translation and is a long-term organizer of [WMT](/events/wmt.md).
```
Comment (cefoo) [inline people/ondrej-bojar.md:10]: The WMT link should be `[WMT](/../events/wmt.md)`

File: people/ondrej-bojar.md [hunk 7]
```diff
@@ -5,8 +5,8 @@ title: Ondřej Bojar
...
 
 ## Research topics
```
Comment (cefoo) [inline people/ondrej-bojar.md:12]: `[post-editing](/../workflows/post-editing.md)`

**Comment by zouharvi** [comment]
Hi @cefoo, thanks for checking the links. Fixed.

**Pull request merged**